build/Targets/Imports.targets

```diff
@@ -28,20 +28,23 @@
         <_CopyProjectReferences>false</_CopyProjectReferences>
         <CopyNuGetImplementations>false</CopyNuGetImplementations>
         <OutputPath>$(OutputPath)Dlls\$(MSBuildProjectName)\</OutputPath>
+        <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
```

Comment:
We should also do this for VSIX projects.

Reply:
Could you clarify why? 
Ref assemblies are only useful for non-terminal projects. My understanding is that VSIX projects are terminal (no other projects depend on binaries they produce).

Reply:
@jasonmalinowski I'll go ahead and merge. We can add the same for VSIX if you still think that has a purpose. Thanks